x/emissions/module/rewards_v2.go

```diff
@@ -0,0 +1,44 @@
+package module
+
+import (
+	"fmt"
+	"math"
+)
+
+// GetWorkerScore calculates the worker score based on the losses and lossesCut.
+// Consider the staked weighted inference loss and one-out loss to calculate the worker score.
+// T_ij / T_ik / T^-_ik / T^+_ik
+func GetWorkerScore(losses, lossesCut float64) float64 {
+	deltaLogLoss := math.Log10(lossesCut) - math.Log10(losses)
+	return deltaLogLoss
+}
+
+// GetStakeWeightedLoss calculates the stake-weighted average loss.
+// Consider the losses and the stake of each reputer to calculate the stake-weighted loss.
+// The stake weighted loss is used to calculate the network-wide losses.
+// L_i / L_ij / L_ik / L^-_i / L^-_il / L^+_ik
+func GetStakeWeightedLoss(reputersStakes, reputersReportedLosses []float64) (float64, error) {
```

Comment:
no types on the first arg? same for function above

Reply:
Omitting the type of all but the last parameter in a list of parameters with the same type in Go indicates that these parameters share the same type as the one specified.